kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue

```diff
@@ -256,10 +256,52 @@
         });
       },
     },
+    watch: {
+      taskPercentage(newPercentage) {
+        if (newPercentage !== null) {
+          const formattedPercentage = (newPercentage * 100).toFixed(2) + '%';
```

Comment:
We will need to internationalize the text here, the `$formatNumber` method is available on `this` to achieve this. See here for more details https://github.com/learningequality/vue-intl?tab=readme-ov-file#formatnumber, including an example of formatting a number as a percentage.